.github/workflows/packages.yml

```diff
@@ -51,16 +50,15 @@ jobs:
           --health-timeout 5s
           --health-retries 5
 
-
```

Comment:
You don't like two spaces? 🥺

Reply:
I forgot to mention. `prettier` change the formatting of the file. I will changed it.